src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -192,6 +194,31 @@ public String getInstanceCapStr() {
         }
     }
 
+    public void setIdleMinutes(int i) {
+        this.idleMinutes = i;
+    }
+
+    public int getIdleMinutes() {
+        return idleMinutes;
+    }
+
+    @DataBoundSetter
+    public void setIdleMinutesStr(String idleMinutes) {
+        if ("".equals(idleMinutes)) {
```

Comment:
Is it ok to introduce a dependency here? I was just trying to be consistent with what is implemented for setInstanceCapStr.